persistence/sql/identity/persister_identity.go

```diff
@@ -649,26 +644,38 @@ func QueryForCredentials(con *pop.Connection, where ...Where) (map[uuid.UUID](ma
 	return credentialsPerIdentity, nil
 }
 
-func (p *IdentityPersister) ListIdentities(ctx context.Context, params identity.ListIdentityParameters) (res []identity.Identity, err error) {
+func (p *IdentityPersister) ListIdentities(ctx context.Context, params identity.ListIdentityParameters) (_ []identity.Identity, nextPage *keysetpagination.Paginator, err error) {
 	ctx, span := p.r.Tracer(ctx).Tracer().Start(ctx, "persistence.sql.ListIdentities")
 	defer otelx.End(span, &err)
 
 	span.SetAttributes(
-		attribute.Int("page", params.Page),
-		attribute.Int("per_page", params.PerPage),
+		// attribute.Int("page", params.Page),
+		// attribute.Int("per_page", params.PerPage),
 		attribute.StringSlice("expand", params.Expand.ToEager()),
 		attribute.Bool("use:credential_identifier_filter", params.CredentialsIdentifier != ""),
 		attribute.String("network.id", p.NetworkID(ctx).String()),
 	)
 
+	paginator := keysetpagination.GetPaginator(append(
+		params.KeySetPagination,
+		keysetpagination.WithDefaultToken(identity.DefaultPageToken()),
+		keysetpagination.WithDefaultSize(250),
+		keysetpagination.WithColumn("id", "ASC"))...)
+
 	is := make([]identity.Identity, 0)
 
 	con := p.GetConnection(ctx)
 	nid := p.NetworkID(ctx)
 
 	joins := ""
-	wheres := ""
-	args := []any{nid}
+	wheres := "identities.nid = ? AND identities.id > ?"
+	args := []any{nid, paginator.Token().Encode()}
+	limit := fmt.Sprintf("LIMIT %d", paginator.Size()+1)
+	if params.PagePagination != nil {
+		wheres = "identities.nid = ?"
+		args = []any{nid}
+		limit = fmt.Sprintf("LIMIT %d OFFSET %d", params.PagePagination.ItemsPerPage, params.PagePagination.Page*params.PagePagination.ItemsPerPage)
```

Comment:
This might change existing behavior.
Please use `pop.NewPaginator(params.Page+1, params.PerPage)` like we had previously to calculate the same limit and offset.

Reply:
Wait, I thought we had changed to page 0 being the first page?

Reply:
Yes, but pop starts at 1, so our first one `0` +1 is pop's first one `1`. Because this was already breaking customers a few weeks back, I'd like to keep the same logic and use the pop helper to calculate limit & offset, just to make sure there is no breaking behavior again.

Reply:
OK, I think I got it now 😁